examples/line2D_with_Sensitivity/slope_likelihood.C

```diff
@@ -0,0 +1,82 @@
+#include <cmath>
+#include <fstream>
+#include <iostream>
+
+#include <queso/GenericScalarFunction.h>
+#include <queso/GslVector.h>
+#include <queso/GslMatrix.h>
+#include <queso/UniformVectorRV.h>
+#include <queso/StatisticalInverseProblem.h>
+#include <queso/ScalarFunction.h>
+#include <queso/VectorSet.h>
+
+#include <slope_likelihood.h>
+
+template<class V, class M>
+Likelihood<V, M>::Likelihood(const char * prefix,
+	const QUESO::VectorSet<V, M> & domain)
+      : QUESO::BaseScalarFunction<V, M>(prefix, domain),
+	x_vals(0),
+	y_obs(0),
+	stdDevs(0)
+{
+  double xs[21], ys[21], sigmas[21];
+  std::ifstream xs_file("xs.txt");
```

Comment:
It's good practice to check your input files exist and open successfully before reading from them.